templates/OpenEdx/0/docker-compose.yml

```diff
@@ -0,0 +1,125 @@
+db:
+    image: mysql:5.7
+    labels:
+        io.rancher.sidekicks: dbConfig
+    environment:
+        MYSQL_ROOT_PASSWORD: password
+        MYSQL_DATABASE: edxapp
+        MYSQL_USER: edxapp001
+        MYSQL_PASSWORD: password
+    volumes:
+     - ${EDX_MYSQL_DATA_ROOT}:/var/lib/mysql
+    volume_driver: ${volume_driver}
+    
+dbConfig:
+    image: mysql:5.7
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    command: |
+      bash -c "sleep 2m; mysql -uroot -ppassword -h db -e \"create database edxapp_csmh;\""
+
+mongo:
+    labels:
+        io.rancher.sidekicks: mongoConfig
+    image: mongo:3.3
+    volumes:
+     - ${EDX_MONGO_DATA_ROOT}:/data/db
+    volume_driver: ${volume_driver}
+
+mongoConfig:
+    image: mongo:3.3
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    command: |
+      bash -c "sleep 2m; echo 'db.createUser({ user: \"edxapp\", pwd: \"password\", roles: [ { role: \"readWrite\", db: \"edxapp\" } ] });' > createEdxappUser.js; mongo mongo/edxapp createEdxappUser.js; echo 'db.createUser({ user: \"cs_comments_service\", pwd: \"password\", roles: [ { role: \"readWrite\", db: \"cs_comments_service\" } ] });' > createForumUser.js; mongo mongo/cs_comments_service createForumUser.js"
+    
+# Need to build our own for ES 0.9
+es:
+    image: opensaas/edx-elasticsearch
+    volumes:
+     - ${EDX_ES_DATA_ROOT}:/data
+    volume_driver: ${volume_driver}
+
+memcache:
+    image: memcached:1.4.24
+    volumes:
+     - ${EDX_MEMCACHE_DATA_ROOT}:/data
+    volume_driver: ${volume_driver}
+
+
+rabbitmq:
+    image: rabbitmq:3.5.3
+    volumes:
+     - ${EDX_RABBIT_DATA_ROOT}:/var/lib/rabbitmq
+    volume_driver: ${volume_driver}
+    environment:
+        RABBITMQ_DEFAULT_USER: celery
+        RABBITMQ_DEFAULT_PASS: celery
+
+forums:
+    image: opensaas/edx-forums
+    labels:
+        io.rancher.container.pull_image: always
+    volumes:
+     - ${EDXAPP_CS_COMMENTS_ROOT}:/edx/app/forum/cs_comments_service
+    volume_driver: ${volume_driver}
+   
+
+lms:
+    labels:
+        io.rancher.container.pull_image: always
+    image: opensaas/edxapp:trusty-v3

```

Comment:
@leodotcloud the trusty-v3 denotes the edxops base image that was used to build the image (ie Ubuntu Trusty). Happy to tag it with something else if you think it will make more sense..